src/RoleObjects/AccessibilityObject.js

```diff
@@ -151,23 +150,141 @@ export default class AccessibilityObject {
   }
 
   /**
-   * Sets the keyboard shortcut to click the DisplayObject
+   * The translate attribute specifies whether the content of an element should be translated or not.
```

Comment:
Looks like the comments for the accessKey getter and setter were swapped with that of the translate getter and setter

Reply:
Done